bin/release

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env ruby
+
+require_relative "../lib/local_time/version"
+require "pathname"
+require "json"
+
+class Release
+  def initialize(version, dry: false)
+    @version = version
+    @dry = dry
+  end
+
+  def perform
+    write_version
+    build_gem
+
+    unless dry?
+      publish
+      cleanup
+    end
+
+    puts "== Done! =="
+  end
+
+  private
+    attr_reader :version, :dry
+
+    def write_version
+      puts "== Writing version to files =="
+
+      if Gem::Version.new(version) <= Gem::Version.new(LocalTime::VERSION)
+        abort "❌ New version (#{version}) must be greater than #{LocalTime::VERSION}"
```

Comment:
I think you have to bundle after updating the version so that it propagates to the gemfile. Not totally sure, though. In any case, if this works, perfect.

Reply:
Ah, we didn't use to have the gem's version in the lock file because `gemspec` was missing from the `Gemfile`.

Added here, and cleaned up some dependencies: https://github.com/basecamp/local_time/pull/138/commits/c0cbb5675506c122a669e87f8963e4a2adcfd56c